tools/docker/tests/test_flows.sh

```diff
@@ -98,8 +99,12 @@ test_ap_capability_info_reporting() {
 }
 
 test_client_capability_info_reporting() {
-    #TODO: Implement
-    return 1
+#TODO: Implement    
+    status "test client capability"   
+    eval send_bml_command '"bml_wfa_ca_controller \"DEV_SEND_1905,DestALid,$AL_MAC,MessageTypeValue,0x8009,tlv_type,0x90,tlv_length,0xC,tlv_value,{$RADIO_WLAN0_MAC 0x000000110022}\""' $redirect
+    sleep 1
+    dbg "Confirming client capability query has been received on agent"
+    docker exec -it repeater sh -c 'grep -i -q "CLIENT CAPABILITY" /tmp/$USER/beerocks/logs/beerocks_agent_wlan0.log'
```

Comment:
Also, check that the other agent received it. Currently, both will just print `CLIENT CAPABILITY` since they have the same handle, but in the future, we will add a check to verify if the `BSSID == radio mac`.